### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SweetVishnya**
P.S. Very neat looking new site, btw)))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,17 +140,17 @@ packages for Visual Studio 2012.
 
 Triton is strongly powered by [Quarkslab](https://quarkslab.com) for years but also by several strong contributors:
 
-* **Alberto Garcia Illera** - Salesforce
-* **Alexey Vishnyakov** - ISP RAS
+* **Alberto Garcia Illera** - Cruise Automation
```
Comment (JonathanSalwan) [inline README.md:143]: I think alberto was at Salesforce when committing for Ponce. (cc @illera88)

File: README.md [hunk 2]
```diff
@@ -140,17 +140,17 @@ packages for Visual Studio 2012.
...
-* **Daniil Kutz** - ISP RAS
+* **Daniil Kuts** - ISP RAS
 * **Florent Saudel** - Bordeaux University
 * **Jessy Campos** - n/a
-* **Jonathan Salwan** - Quarkslab
+* [**Jonathan Salwan**](http://shell-storm.org/) - Quarkslab
 * **Pierrick Brunet** - Quarkslab
 * **PixelRick** - n/a
-* **Romain Thomas** - Quarkslab
+* [**Romain Thomas**](https://www.romainthomas.fr/) - Quarkslab
```
Comment (JonathanSalwan) [inline README.md:154]: If we highlight people on this list with link we have to do it for every one (for example with @githhub link if no website/twitter exist.

File: README.md [hunk 3]
```diff
@@ -193,7 +194,7 @@ Triton is strongly powered by [Quarkslab](https://quarkslab.com) for years but a
 </li>
 <li>
 <b>Sydr: Cutting Edge Dynamic Symbolic Execution</b><br />
- <b>Talk at</b>: Ivannikov ISP RAS Open Conference, Moscow, Russia, 2020. [<a href="publications/ISPRAS2020-sydr.pdf">paper</a>]<br />
+ <b>Talk at</b>: Ivannikov ISP RAS Open Conference, Moscow, Russia, 2020. [<a href="publications/ISPRAS2020-sydr.pdf">paper</a>] [<a href="https://vishnya.xyz/vishnyakov-isprasopen2020.pdf">slide</a>] [<a href="https://www.ispras.ru/conf/2020/video/compiler-technology-11-december.mp4#t=6021">video</a>]<br />
```
Comment (JonathanSalwan) [inline README.md:197]: Can you push your slides on the `publications` directory?

File: README.md (hunk 1 above)
Comment (illera88) [reply README.md:143]: I was

File: README.md (hunk 2 above)
Comment (SweetVishnya) [reply README.md:154]: I found everyone except Florent Saudel. Do you have any links?

Comment (JonathanSalwan) [reply README.md:154]: Looks like he disappeared from the internet!

Comment (SweetVishnya) [reply README.md:154]: Anyway, I just fixed all other issues and added new papers.

**Pull request merged**